.travis.yml

```diff
@@ -1,22 +1,29 @@
-#******************************************************************************
- # Copyright 2015 Capital One Services, LLC
- 
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at
- 
- # http://www.apache.org/licenses/LICENSE-2.0
- 
- # Unless required by applicable law or agreed to in writing, software
- # distributed under the License is distributed on an "AS IS" BASIS,
- # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- # See the License for the specific language governing permissions and
- # limitations under the License.
- #
- #******************************************************************************/
- 
 language: objective-c
```

Comment:
Can we add the license portion back? I know that Capital One is too particular over this.